### ISSUE TITLE
Continually calling Api.Text in a loop - eventually crashes.

### ISSUE THREAD

**Issue by dazinator** [halkar/Tesseract.Xamarin#5]
My algorithm does this:

1. Initialises the API
2. Begins an endless loop with the body:
``` csharp
                           _Api.SetImage(filePath)
                            .ContinueWith((t) =>
                            {
                                RunOnUiThread(() =>
                                    {
                                        _OcrText.Text = _Api.Text;
                                    });
                            });
```

After a few seconds of execution, there is an exception tfrom within the  `_Api.Text` method. Do you have any existing test coverage for this scenario - i.e one that calls _API.Text many times on the same API instance?

**Comment by halkar** [comment]
No, I haven't implemented any tests for the lib yet. Shame on me. So all my testing is pure manual. Could you please give more extended sample of your code as at the moment it work's fine for me.

**Comment by dazinator** [comment]
Sure - here is the complete android activity I am running!

https://gist.github.com/dazinator/e0ee587676a6ff454e9b

**Comment by dazinator** [comment]
- and I also just added the corresponding `axml` layout to that gist.

**Comment by halkar** [comment]
I created sample for camera preview recognition https://github.com/halkar/Tesseract.Xamarin/tree/android-camera-preview-sample
I managed to work directly with byte array received from the camera without saving data to file.
The only problem is that I don't know how to work with camera preview. I'm a little bit lazy, so I decided not to use queue but just to skip images received while recognitions is working.
So I found one problem - calling SetImage during recognition crashes the application - so for now avoid that, I'll try to fix it the next update.
